src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -272,9 +272,15 @@
 
   @join assert : test.asserts
     @if assert.hasExpectedValueTransformFunction
-      Assert.assertEquals({@assert.expectedValueTransformFunction}(\
-        {@assert.expectedValueIdentifier}), \
-        actualRequest.{@assert.actualValueGetter}());
+      @if assert.hasActualValueTransformFunction
```

Comment:
Yes, I didn't add the case because it was not possible to get to that case (plus it would make the snip file look worse). When you say comment what exactly do you mean (comments in snip file, if snip files support comments at all, or a comment in generated java code)?